### ISSUE TITLE
Unclear how to install libspotify.framework on OSX

### ISSUE THREAD

**Issue by dsiah** [FrontierPsychiatrist/node-spotify#76]
Hi, pretty simple question -- I would like to install the libspotify in the correct place so that I stop getting 
```
../src/Application.h:4:10: fatal error: 'libspotify/api.h' file not found
#include <libspotify/api.h>
```
this error on npm installation.

I was able to install the actual C library (I placed the framework in the directory that I was working) and compile a simple hello.c file with gcc (I am using bash) however I do not know how/where the npm installation is looking for libspotify/api.h.

I am unable to install libspotify using brew.

**Comment by elliotboney** [comment]
In case some1 else runs into this problem, I had to run this for i could ```npm install```
```bash
sudo ln -s /Library/Frameworks/libspotify.framework/libspotify /usr/local/lib/libspotify.dylib
```

**Comment by rmarscher** [comment]
I had a similar issue with `'libspotify/api.h' file not found` on OSX Yosemite with XCode 6.3.1, homebrew 0.9.5, node-gyp 1.0.3, and node 0.10.38.

I installed libspotify via homebrew. `/usr/local/include/libspotify/api.h` exists as does `/usr/local/lib/libspotify.dylib`. I don't know what node-gyp isn't finding it.

I modified binding.gyp and inserted `"/usr/local/opt/libspotify/include"` to the `"include_dirs"` array.

I'm not sure if that is a viable fix that I should submit a pull request for. It's just one line anyway - https://github.com/rmarscher/node-spotify/commit/72787cf3d0bf039b08ac4cd2b4a5662632b71654

It may be possible something is wrong with my system that is causing it to not look in `/usr/local` as it should.
